store/sqlstore/thread_store.go

```diff
@@ -110,7 +110,7 @@ func (s *SqlThreadStore) get(ex gorp.SqlExecutor, id string) (*model.Thread, err
 	err := ex.SelectOne(&thread, query, args...)
 	if err != nil {
 		if err == sql.ErrNoRows {
-			return nil, store.NewErrNotFound("Thread", id)
```

Comment:
There are some places in the post store and the app where where we do a `errors.As` check for not found. I dont think this change will break those usages, but we should double check.

Reply:
I'll double check those

Reply:
Checked and we're not using it for GetThread anywhere